src/main/java/org/broadinstitute/hellbender/tools/spark/sv/CollectInsertSizeMetricsSpark.java

```diff
@@ -252,21 +258,21 @@ public boolean test(final GATKRead read){
     }
 
     @VisibleForTesting
-    void writeMetricsFile(final InsertSizeMetricsCollectorSpark collector, final SAMFileHeader readsHeader, final String inputFileName){
+    void writeMetricsFile(final InsertSizeMetricsCollectorSpark collector, final String inputFileName){
 
-        final MetricsFile<InsertSizeMetrics, Integer> metrics = getMetricsFile();
+        final MetricsFile<InsertSizeMetrics, Integer> metricsFile = getMetricsFile();
 
-        collector.produceMetricsFile(metrics);
+        collector.produceMetricsFile(metricsFile);
 
-        MetricsUtils.saveMetrics(metrics, OUTPUT, getAuthHolder());
+        MetricsUtils.saveMetrics(metricsFile, OUTPUT, getAuthHolder());
```

Comment:
Not a comment against your code but it seems wrong for your tool to have call getAuthHolder() just to save metrics. I think this should be abstracted away, at least to MetricsUtils. Let's discuss with the GATK4 team if we can make this easier. (Don't need to fix in this PR though).